packages/editor/src/bindings/post-meta.js

```diff
@@ -16,13 +16,64 @@ export default {
 		return args.key;
 	},
 	getValue( { registry, context, args } ) {
+		const postId = context.postId
+			? context.postId
+			: registry.select( editorStore ).getCurrentPostId();
 		const postType = context.postType
 			? context.postType
 			: registry.select( editorStore ).getCurrentPostType();
 
 		return registry
 			.select( coreDataStore )
-			.getEditedEntityRecord( 'postType', postType, context.postId )
-			.meta?.[ args.key ];
+			.getEditedEntityRecord( 'postType', postType, postId ).meta?.[
+			args.key
+		];
+	},
+	setValue( { registry, context, args, value } ) {
+		const postId = context.postId
+			? context.postId
+			: registry.select( editorStore ).getCurrentPostId();
+		const postType = context.postType
+			? context.postType
+			: registry.select( editorStore ).getCurrentPostType();
+		registry
+			.dispatch( coreDataStore )
+			.editEntityRecord( 'postType', postType, postId, {
+				meta: {
+					[ args.key ]: value,
+				},
+			} );
+	},
+	lockAttributesEditing( { select, context = {}, args } ) {
+		const postId = context.postId
+			? context.postId
```

Comment:
Is there any special reason of initialising context s an object rather than doing `context?.postId` ?

Reply:
No reason, I can change that 🙂